recipe/build.sh

```diff
@@ -3,5 +3,5 @@
 set -o xtrace -o nounset -o pipefail -o errexit
 
 unset STRIP
-make AMALGAMATED=1
-make install PREFIX=${PREFIX}
+make AMALGAMATED=1 CFLAGS=-D_BSD_SOURCE
+make install PREFIX=${PREFIX} CFLAGS=-D_BSD_SOURCE
```

Comment:
```suggestion
make AMALGAMATED=1 CFLAGS="${CFLAGS} -D_BSD_SOURCE"
make install PREFIX=${PREFIX} CFLAGS="${CFLAGS} -D_BSD_SOURCE"
```
Let's use this so that we don't lose the old `CFLAGS`.